<launch>

<node name="mbed_interface" pkg="cr17" type="mbed_interface.py" respawn="true" launch-prefix="sudo -E "  />

</launch>
